make/RunTests.gmk

```diff
@@ -869,12 +869,7 @@ define SetupRunJtregTestBody
 
   $1_JTREG_BASIC_OPTIONS += -e:TEST_IMAGE_DIR=$(TEST_IMAGE_DIR)
 
-  ifneq ($$(DOCS_IMAGE_DIR), )
-    DOCS_PATH := $$(DOCS_IMAGE_DIR)
-  else
-    DOCS_PATH := $$(OUTPUTDIR)/images/docs
-  endif
-  $1_JTREG_BASIC_OPTIONS += -e:DOCS_IMAGE_DIR=$$(DOCS_PATH)
+  $1_JTREG_BASIC_OPTIONS += -e:DOCS_JDK_IMAGE_DIR=$$(DOCS_JDK_IMAGE_DIR)
```

Comment:
I didn't include a null check here as `DOCS_JDK_IMAGE_DIR` will always hold a value (e.g `build/.../images/docs`) even if the docs don't actually exist. I will be up to the tests to skip when docs aren't found.

Reply:
It's fine to leave out the conditional, but the double `$$` isn't needed and I'm surprised if it actually works. That must be some lucky double evaluation somewhere in that case. 

```suggestion
  $1_JTREG_BASIC_OPTIONS += -e:DOCS_JDK_IMAGE_DIR=$(DOCS_JDK_IMAGE_DIR)
```

Reply:
I'm not sure why you are saying that. This is in the body of `SetupRunJtregTest`, which will be eval:ed when called, as usual. So `$$` should be used for all variable expansions. It's actually the other way around; `$` will only work due to pure "luck" in this case, and we kind of mis-used that before, splitting the set of variables into two parts, one there `$` would suffice, and one where `$$` were needed. This was just confusing, to save an extra keypress, so I've tried to use `$$` everywhere and always in these macro bodies, for consistency.

Reply:
Oh, I misread the context. Magnus is correct.